lib/config/definitions.ts

```diff
@@ -94,6 +94,18 @@ const options: RenovateOptions[] = [
     default: [],
     admin: true,
   },
+  {
+    name: 'ansible-galaxy',
```

Comment:
We refactored how managers are added so there's no need for this block now. The manager should just export a `defaultConfig` in its `index.ts` file.